benchmarks/functional_autograd_benchmark/functional_autograd_benchmark.py

```diff
@@ -0,0 +1,144 @@
+import torch
+from torch.autograd import functional
+
+import time
+import argparse
+from collections import namedtuple, defaultdict
+
+import ppl_models
+import vision_models
+import audio_text_models
+
+from utils import get_str
+
+# Listing of the different tasks
+FAST_TASKS_NO_DOUBLE_BACK = [
+    "vjp",
+]
+
+FAST_TASKS = FAST_TASKS_NO_DOUBLE_BACK + [
+    "vhp",
+    "jvp",
+]
+
+ALL_TASKS = FAST_TASKS + [
+    "hvp",
+    "jacobian",
+    "hessian"
+]
+
+DOUBLE_BACKWARD_TASKS = ["jvp", "hvp", "vhp", "hessian"]
+
+# Model definition which contains:
+# - name: a string with the model name.
+# - getter: a function to get the mode. It takes as input the device on which the model
+#     will run. It should return the forward function and the parameters (Tensors) used as
+#     input for the forward function. Note that the forward must *not* have any side effect.
```

Comment:
"a function to get the mode" -> "a function to get the model"?

If we add a type annotation for this, then we can clearly annotate `getter` taking a `torch.device` and returning a Callable and some inputs